extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelContextProcessor.java

```diff
@@ -163,4 +168,50 @@ public CamelRuntimeBuildItem runtime(
                 recorder.createRuntime(beanContainer.getValue(), context.getCamelContext()),
                 config.bootstrap.enabled);
     }
+
+    /**
+     * Registers Camel CDI event bridges if quarkus.camel.event-bridge.enabled=true and if
+     * the relevant events have CDI observers configured for them.
+     *
+     * @param beanDiscovery build item containing the results of bean discovery
+     * @param context       build item containing the CamelContext instance
+     * @param recorder      the CamelContext recorder instance
+     */
+    @Record(ExecutionTime.STATIC_INIT)
+    @BuildStep(onlyIf = EventBridgeEnabled.class)
+    public void registerCamelEventBridges(
+            BeanDiscoveryFinishedBuildItem beanDiscovery,
+            CamelContextBuildItem context,
+            CamelContextRecorder recorder) {
+
+        Set<String> observedLifecycleEvents = beanDiscovery.getObservers()
+                .stream()
+                .map(observerInfo -> observerInfo.getObservedType().name().toString())
+                .filter(observedType -> observedType.startsWith("org.apache.camel.quarkus.core.events"))
+                .collect(Collectors.collectingAndThen(Collectors.toUnmodifiableSet(), HashSet::new));
+
+        Set<String> observedManagementEvents = beanDiscovery.getObservers()
+                .stream()
+                .filter(observerInfo -> observerInfo.getObservedType().name().toString()
+                        .matches("org.apache.camel(?!.quarkus).*Event$"))
+                .map(observerInfo -> observerInfo.getObservedType().name().local())
```

Comment:
> I still wonder why we cannot use FQ names here?

Because in theory for Camel management events someone may code their app to either observe on the interface type. E.g `org.apache.camel.spi.CamelEvent.ExchangeCompletedEvent` or on the implementation class `org.apache.camel.impl.event.ExchangeCompletedEvent`. 

In that case we need to match on the simple name, because the event type receieved at runtime is always the implementation class. Hence the usage of `local()` to avoid additional bits like '$' etc.

Maybe we should not support observing on the plain interface type and we just mandate observing on the impl classes?

Reply:
OK, thanks, now I see that matching on simple name allows to generalize from interface name in the handler (build time) declaration to an impl class name when filtering at runtime.

The more I think `DotName.local()` is not what we need. `myDotName.local()` returns the simple name `MyClass` only if `myDotName` was created via `Dotname.createComponentized("org.myorg.ns", "MyClass")`. 
But that's not the only option. If it was created via `Dotname.createSimple("org.myorg.ns.MyClass")` then `local()` returns the whole `"org.myorg.ns.MyClass"`. 
Because we never know how the supplied DotNames got created, we can never know whether `local()` returns a simple name or a FQ name.

Reply:
I tweaked things a bit to obtain an instance of the observed event class and then use it to get the simple name. Hopefully that's acceptable.

I also added some comments explaining why simple name is used.